jhotdraw-core/jgiven-reports/org.jhotdraw.draw.figure.BringToFrontActionTest.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
I don't think JGiven reports are supposed to be included?

Reply:
Reports should be saved for logging and auditing purposes

Reply:
The result of the test may change over time, as this is *only* updated on manual runs.

Reply:
Since they are changed over time it is valuable to store in version control with their respective state. 
They should also be saved as artifacts during CI.

Reply:
I do not believe that they should be stored in Git.  
But it might be debatable if they should be uploaded as artifacts in GitHub action.

Reply:
The files are small and isolated in their own repository. 
If this becomes an issue later on, we can find another solution at that time. Let's keep it as simple as possible to begin with.